CRM/Member/Form/MembershipBlock.php

```diff
@@ -160,8 +160,7 @@ public function buildQuickForm() {
         $this->assign('auto_renew', $this->_renewOption);
       }
       $this->addGroup($membership, 'membership_type', ts('Membership Types'));
-      $this->addGroup($membershipDefault, 'membership_type_default', ts('Membership Types Default'))
-        ->setAttribute('allowClear', TRUE);
+      $this->addGroup('membership_type_default', ts('Membership Types Default'), $membershipDefault, ['allowClear' => TRUE]);
```

Comment:
Here too. Should be addRadio.

Calling it a day so no hurry.

Reply:
should be sorted now